### DISCUSSION TITLE
What changes or additions would you like to see?

### DISCUSSION THREAD

**Discussion by jprochazka** [jprochazka/adsb-receiver#557] [Polls]
I am looking for suggestions from the community as to what you would like to see added or modified in the next version. Maybe a replacement for a removed feature, updates to a current one, support for distributions previously supported, or a totally new feature never available before.

In the latest release you will notice a few options missing that were available in previous versions due to the fact the code bases for said items have been abandoned resulting in failed builds due to outdated code on newer distributions. You may wish to look over the current [CHANGELOG](https://github.com/jprochazka/adsb-receiver/blob/master/CHANGELOG.md) in the phoenix branch in order to get an idea of what might be missing from older releases. 

I look forward to viewing your feedback!

**Comment by cptacek** [comment]
Joe, Happy to see your project getting back on its feet. Curious if you have taken a look at https://adsb.im/ . Sounds like you're taking your project in this type of direction. I have no affiliation besides being a user of the software.

**Reply by jprochazka** [reply]
Hello @cptacek good to hear from you!

ADSB.im looks to go in the same direction this project does as far as allowing users to set up a receiver which feeds multiple aggregate sites, but I would say that is where the similarities end. ADSB.im appears to be an all-in-one turnkey solution where this project offers a more customizable more adaptable solution.

The project initially started out about 10 years ago by adding a simple HTML page displaying dump1090 statistics using collectd and dump1090-tools much like you see other projects utilizing now a days, to a base PiAware image.

The direction of the project has since evolved as needs changed. No longer did simply writing an image to an SD card fulfill everyone's needs including myself. The project changed directions to become something which allowed users to easily setup and run ADS-B receiver software on not only on the Raspberry Pi but on other hardware as well, meaning you were not limited to running a pre-made disk image built for a particular hardware brands or architecture. This was accomplished by building each application from source when available or choosing the proper pre-compiled binary when source code is not available. Needless to say, the process is a bit more involved than writing a disk image but rewards the user with a more customize/streamlined installation. 

Although a bit dated and in need of updates I feel the portal sets this project apart as well. It does not only contain dump1090 statistics but is also capable of retaining flight data for historical purposes if one so desires among other things.

**Comment by YogySK** [comment]
Hello, I am glad this project is coming back to life!

I was wondering if it would be possible to implement OGN (Open Glider Network) decoder in this install?

And thank you for your time invested in this project and for the work/research done!

**Reply by jprochazka** [reply]
I do not see any reason why an installation option could not be made available as soon as this upcoming weekend if not sooner. I first need to look into a possible Graphs1090 compatibility issue with the original portal's graphs for another member of the community.

One thing to keep in mind will be the fact I am located within the United States where such data is generally not available. Looking at the OGN flight tracking site I see there are a handful of tracks across Lake Erie in Canada but due to the nature of the aircraft using this technology I cannot guarantee much support for this option beyond installation. Those looking for support beyond installation will need to seek support from OGN or the community.

**Comment by jprochazka** [comment]
I am going to place this thought within this thread for feedback from the community. I am honestly not sure that uninstallation options are actually necessary and worth spending time on. I will leave it up to you to help determine when such an option should be made available. Simply put if people are not uninstalling this time can be better spent on other features.

Lately installation options have been added which are not being built as Debian packages. This is not a bad thing in fact building all applications in the future using something as simple as make/make install could result in support for non-Debian distributions as well. The only problem is the fact it is harder to keep track of what was installed/created/utilized by these types of installations. An uninstallation option for each item may be necessary in order to make the removal of options a user may no longer wish installed easier by removing all associated items while not breaking others.
